### PR TITLE
Added integration test for Stackdriver Storage with autoconfiguration

### PR DESCRIPTION
**Pull request by saturnism**
For #130 

Added integration test for Zipkin Stackdriver Reporter, which is a different path than the Stackdriver Sender.

### CODE REVIEW

**Comment by saturnism** [comment]
Good thing I'm doing this w/ a PR outside of `zipkin-gcp` repo ;) caught quite a few issues w/ travis and PR. Fixed those in this commit too.

File: pom.xml [hunk 1]
```diff
@@ -49,9 +49,10 @@
     <zipkin.version>2.16.2</zipkin.version>
     <zipkin-reporter.version>2.10.2</zipkin-reporter.version>
     <brave.version>5.6.10</brave.version>
-    <grpc.version>1.22.1</grpc.version>
+    <grpc.version>1.22.2</grpc.version>
```
Comment (saturnism) [inline pom.xml]: this is updated to match armeria's grpc version.

File: .travis.yml [hunk 2]
```diff
@@ -18,7 +18,8 @@ before_install:
   # Encrypted service account file for integration testing
   # This file is referenced via GOOGLE_APPLICATION_CREDENTIALS env var
   # Ex. travis encrypt-file travis/zipkin-gcp-ci-0d7917f58da7.json
-  - openssl aes-256-cbc -K $encrypted_0d0e64b78c84_key -iv $encrypted_0d0e64b78c84_iv -in travis/zipkin-gcp-ci-0d7917f58da7.json.enc -out travis/zipkin-gcp-ci-0d7917f58da7.json -d
```
Comment (elefeint) [inline .travis.yml]: In spring-cloud-gcp, we [explicitly gate](https://github.com/spring-cloud/spring-cloud-gcp/blob/0513b8784f8828918aacacf18db80a992ae4134c/.travis.yml#L24) on $TRAVIS_SECURE_ENV_VARS. In principle, TRAVIS_PULL_REQUEST could also be used to conditionally decrypt as it's false when not running on a pull request.

Environment variable reference: https://docs.travis-ci.com/user/environment-variables/

File: sender-stackdriver/pom.xml [hunk 3]
```diff
@@ -98,7 +98,7 @@
     <dependency>
       <groupId>org.awaitility</groupId>
       <artifactId>awaitility</artifactId>
-      <version>3.1.6</version>
+      <version>${awaitability.version}</version>
```
Comment (elefeint) [inline sender-stackdriver/pom.xml]: s/awaitability/awaitility

Comment (saturnism) [reply sender-stackdriver/pom.xml]: good catch!

File: .travis.yml (hunk 2 above)
Comment (saturnism) [reply .travis.yml]: great idea; i'll do the same.

Comment (saturnism) [reply .travis.yml]: there is a similar gate for decrypting file: https://github.com/spring-cloud/spring-cloud-gcp/blob/0513b8784f8828918aacacf18db80a992ae4134c/.travis.yml#L37

i've updated here to use the same gate logic.

**Pull request merged**